docs/index.md

```diff
@@ -0,0 +1 @@
+Welcome to Pugbot documentation!
```

Comment:
Can we have a link to the commands?
Might be your style but your example, https://rayneh.github.io/Pubobot/, doesn't seem to have a link to the commands.